portfolio/src/main/webapp/script.js

```diff
@@ -32,16 +32,34 @@ function funFactGenerator() {
   factGetter.innerText = fact;
 }
 
-function serverFunFact() {
-    
+/**
+ * Displays a fun fact about Desmond on the page.
+ */
+function serverFunFact() { 
     fetch('/data').then(facts => facts.json()).then(jsonfacts => {
 
-    // Pick a random fact.
-    const fact = jsonfacts[Math.floor(Math.random() * jsonfacts.length)];
+        // Pick a random fact.
+        const fact = jsonfacts[Math.floor(Math.random() * jsonfacts.length)];
+
+        // Add it to the page.
+        const factGetter = document.getElementById('factgetter');
+        factGetter.innerText = fact;
+    })
+}
+
+/**
+ * Adds user's comment to the page.
+ */
+function showComments() {
 
-    // Add it to the page.
-    const factGetter = document.getElementById('factgetter');
-    factGetter.innerText = fact;
+    // Starts post request using user's input.
+    var init = {method: "POST",
+                body: document.getElementById("textfield").value};             
+    var request = new Request("/data", init);
 
+    // Fetches server comment data and writes it to the page.
+    fetch(request).then(comments => comments.json()).then(jsoncomments => {
+        document.getElementById("thecomments").innerHTML += "<p>" + 
```

Comment:
do you need tags inside of an element? Not sure how this looks in the UI.

Reply:
Yeah I need to <p> tags so that each comment is on their own line. I could have used "\n" likely but I wanted to leave myself the option to format those comments a little better in CSS later.